gravitee-am-service/src/main/java/io/gravitee/am/service/impl/OrganizationUserServiceImpl.java

```diff
@@ -143,4 +143,20 @@ public Single<User> update(User user) {
                     return Single.error(new TechnicalManagementException("An error occurs while trying to update a user", ex));
                 }));
     }
+
+    @Override
+    public Single<AccountAccessToken> generateAccountAccessToken(User user, NewAccountAccessToken newAccountToken) {
+        var rawToken = SecureRandomString.generate();
+
+        var token = AccountAccessToken.builder()
+                .tokenId(RandomString.generate())
+                .referenceType(ORGANIZATION)
+                .referenceId(user.getReferenceId())
+                .userId(user.getId())
+                .name(newAccountToken.name())
+                .token(accountAccessTokenEncoder.encode(rawToken))
+                .build();
+        return accessTokenRepository.create(token)
+                .map(created -> created.withToken(rawToken));
```

Comment:
issue: we are not linking the token with the Encoder settings, so if the platform admin change the settings (new encoder or another number of rounds) then the existing token will be unusable.